packages/webview_flutter/webview_flutter_platform_interface/lib/src/platform_interface/webview_platform_callbacks_handler.dart

```diff
@@ -24,9 +24,16 @@ abstract class WebViewPlatformCallbacksHandler {
   void onPageFinished(String url);
 
   /// Invoked by [WebViewPlatformController] when a page is loading.
-  /// /// Only works when [WebSettings.hasProgressTracking] is set to `true`.
+  /// Only works when [WebSettings.hasProgressTracking] is set to `true`.
   void onProgress(int progress);
 
+  /// Invoked by [WebViewPlatformController] when the webview's URL has changed.
+  ///
+  /// Unlike [onPageStarted], [onProgress], and [onPageFinished],
+  /// [onUrlChanged] also fires when navigating without a full page load
+  /// e.g. when using a single page application.
```

Comment:
nit: saying "using a single page application" doesn't describe when the `onUrlChanged` callback is called and the `onPageStarted`, `onProgress` and `onPageFinished` not. Changing it too "when navigating within a single page application."  adds a bit more context and makes the example easier to comprehend.

```suggestion
  /// e.g. when navigating within a single page application.
```